src/main/java/com/google/growpod/servlets/UserServlet.java

```diff
@@ -127,4 +131,20 @@ public UserDao getDao() {
   public void setDao(UserDao dao) {
     this.dao = dao;
   }
+
+  /**
+   * Currently this Post function will get the user data in the form of a Json object,turn the Json
+   * object into a User object and call a function that will in Datastore.
+   *
+   * @param req Information about the POST request
+   * @param resp Information about the servlet's response
+   */
+  @Override
+  public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+    Gson gson = new Gson();
+    resp.setContentType("application/json");
+    String jsonString = req.getParameter("userData");
+    User userData = gson.fromJson(jsonString, User.class);
```

Comment:
nit: can inline the Gson constructor here and save a line. e.g.:
`User userData = new Gson().fromJson(jsonString, User.class)`

Reply:
Done!